ee/dtr/admin/configure/external-storage/index.md

```diff
@@ -4,52 +4,54 @@ description: Storage configuration for Docker Trusted Registry
 keywords: storage drivers, NFS, Azure, S3
 ---
 
+## Configure your storage backend
+
 By default DTR uses the local filesystem of the node where it is running to
 store your Docker images. You can configure DTR to use an external storage
 backend, for improved performance or high availability.
 
 ![architecture diagram](../../../images/configure-external-storage-1.svg)
 
-If your DTR deployment only has a single replica, you can continue using the
-local filesystem to store your Docker images. If your DTR deployment has
-multiple replicas, for high availability, you need to ensure all replicas are
-using the same storage backend. When a user pulls an image, the node serving
-the request needs to have access to that image.
+If your DTR deployment has a single replica, you can continue using the
+local filesystem for storing your Docker images. If your DTR deployment has
+multiple replicas, make sure all replicas are
+using the same storage backend for high availability. Whenever a user pulls an image, the DTR
+node serving the request needs to have access to that image.
 
-DTR supports these storage systems:
+DTR supports the following storage systems:
 
 * Local filesystem
-* NFS
-* Amazon S3 or compatible
-* Google Cloud Storage
-* Microsoft Azure Blob storage
-* OpenStack Swift
-
-To configure the storage backend, you can log into the **DTR web UI**
-as an administrator user, navigate to the **Settings** page, and choose
-**Storage**.
+   * NFS
+   * Bind Mount
+   * Volume
+* Cloud Storage Providers
+   * Amazon S3
+   * Microsoft Azure
+   * OpenStack Swift
+   * Google Cloud Storage
+
+To configure the storage backend, log in to the DTR web interface
+as an admin, and navigate to **System > Storage**.
 
 ![dtr settings](../../../images/configure-external-storage-2.png){: .with-border}
 
-The storage configuration page in the DTR web UI has options for the most
-common configuration options, but you can also upload a yaml configuration file.
+The storage configuration page gives you the most
+common configuration options, but you have the option to upload a configuration file in `.yml`, `.yaml`, or `.txt` format.
 
-The format of this configuration file is similar to the one used by
-[Docker Registry](/registry/configuration.md).
+See [Docker Registry Configuration](/registry/configuration.md) for configuration options.
 
 ## Local filesystem
 
 By default, DTR creates a volume named `dtr-registry-<replica-id>` to store
 your images using the local filesystem. You can customize the name and path of
-the volume used by DTR, using the `docker/dtr reconfigure --dtr-storage-volume`
+the volume by using the `docker/dtr reconfigure --dtr-storage-volume`
```

Comment:
if the operator uses `reconfigure --dtr-storage-volume` is there a requirement that they first created that volume on all DTR nodes?  If so, perhaps we should note it here.

Reply:
if ^ is true, we should also note that using `--dtr-storage-volume` creates a future obligation on the operator to create the volume prior to joining nodes that don't have it.  hopefully `join` fails safely if the volume isn't there?

Reply:
Good point - can I also get confirmation on this, @caervs ? Am I misunderstanding this part? 

> If you’re deploying DTR with high-availability, you need to use NFS or any other centralized storage backend so that all your DTR replicas have access to the same images.

It sounds to me like you only have to configure it in one node.

Reply:
create the volume prior to joining nodes that don't have it. - Yeah, good point. Thanks. Will wait on @caervs to confirm before updating this section.

Reply:
DTR installation/reconfigure will automatically create the volume, similar to a `docker run -v`.

However a user should never do this with a multi-node DTR as the volumes would be local.

Reply:
> However a user should never do this [...]

.... unless they have to because they just lost all their tags:

https://success.docker.com/article/dtr-26-lost-tags-after-reconfiguring-storage#reconfigureusingalocalnfsvolume

> 1. On each DTR node, create a local volume pointing to the desired NFS export. [...]